plugins/avx512/float/UMESimdVecFloat64_16.h

```diff
@@ -1501,8 +1501,13 @@ namespace SIMD {
             __m512d t6 = _mm512_castsi512_pd(t4);
             return SIMDVec_f(t5, t6);
 #else
+#if defined (WA_GCC_INTR_SUPPORT_7_1)
+            __m512d t0 = _mm512_abs_pd(_mm512_castpd_ps(mVec[0]));

```

Comment:
Unfortunately this is not correct. _mm512_castpd_ps changes the scalar type from 64 to 32b, and therefore introduces rounding error. The fact that _mm512_abs_pd accepts '_m512' and not '_m512d' is a coding error in the compiler. This was the reason why this code is marked as WA_GCC_INTR_SUPPORT_7_1,

Reply:
__m512 _mm512_castpd_ps (__m512d a)
Cast vector of type __m512d to type __m512. This intrinsic is only used for compilation and does not generate any instructions, thus it has zero latency.

this should not change anything at all of the values. its just  for the user that he's more aware with what kind of values his currently working with. also the tests didnt show any problems with it.